test/03_erc20_enforcer_tests_l1.ts

```diff
@@ -368,19 +368,19 @@ describe('Testing the Holograph ERC20 Enforcer (L1)', async function () {
           .withArgs(l1.deployer.address, l1.wallet2.address, 0);
       });
 
-      it('should fail for non-contract onERC20Received call', async function () {
+      it.skip('should fail for non-contract onERC20Received call', async function () {
```

Comment:
Are these to be reenabled later

Reply:
We removed the erc20 receiver since it’s not a standard or widely used spec. These will be skipped for now and maybe removed entirely if we do not implement an alternative safe transfer function.